src/style/style.css

```diff
@@ -0,0 +1,38 @@
+.form_div {
+	background-color: #354151;
+	height: 100vh;
+	margin: auto;
+	text-align: center;
+	display: block;
+	margin-left: auto;
+	margin-right: auto
+}
+
+.thyme_logo {
+	width: 200px;
+	margin-top: 40px
+}
+
+.user_input {
+	border-radius: 17px;
+	border: 2px solid black;
+	padding-left: 224px;
+	padding-right: 133px;
+	background-color: #354151;
+	padding-top: 6px;
+	padding-bottom: 6px;
+	color : white;
+	margin-top: 20px
+}
+
+.sign_in {
+	border: 2px solid black;
+	border-radius: 17px;
+	padding-left: 235px;
+	padding-right: 236px;
+	background-color: #5cb695;
+	padding-top: 7px;
+	padding-bottom: 7px;
+	color : white;
+	margin-top: 20px
+}
```

Comment:
Why are you using a new css file when we have index.css ?

Reply:
done